### ISSUE TITLE
Issue on page /tune/tutorials/tune-pytorch-lightning.html

### ISSUE THREAD

**Issue by andrei-rusu** [ray-project/ray#17338]
Hello,

I would like to point out that this tutorial looks more like a draft than an actual finalized article. It contains many mistakes, such as:
- referring to the average loss with 2 different names in different sections: 'ptl/val_loss' and 'avg_val_loss' (same goes for the accuracy)
- having confusing code snippets, that refer to completely different things than what the text points to
- having empty code snippets

It is weird no one has noticed these mistakes, they are quite apparent (especially when it comes to the code snippets). Or maybe you do not actively monitor these tutorial pages, in which case a disclaimer should be placed somewhere. Hope it helps :)
